integrations/pr_publish.py

```
@@ -0,0 +1,680 @@
+
+    def whoami(self):
+        return None  # unknown identity → reconcile falls back to marker-only (a dry run mutates nothing)
+
+    def list_issue_comments(self, pr):
+        return []
+
+    def list_review_comments(self, pr):
+        return []
+
+    def create_issue_comment(self, pr, body):
+        self.calls.append(("create_issue_comment", pr, body)); return {"id": 0}
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)